cephfs/errors.go

```diff
@@ -38,4 +38,5 @@ func getError(e C.int) error {
 
 const (
 	errNameTooLong = CephFSError(-C.ENAMETOOLONG)
+	errNoEntry     = CephFSError(-C.ENOENT)
```

Comment:
`errNoDEntry` or `errNoDirEntry` would be better, `errNoEntry` sounds like a permission issue, not like a non-existing (part of a) pathname

Reply:
I wanted it to match ENOENT. Do you think naming it like [errNotExist](https://golang.org/pkg/os/#pkg-variables) like the Go os package is preferable. I do want to use a name we can uppercase (export) in the future if its needed.

Reply:
To me, `errNotExist` sounds the best in that case.

Reply:
Just for the record, for me it seems clear from context, that it's about a file-system entry, and I like the (almost) same naming (principle of least surprise). I'm also fine with `errNotExist`, but shouldn't all errors replicate the Go style then, whenever possible? A mix would be confusing I think. I would consistently stay with one style.

Reply:
Sure, but `man errno` has:

    ENOENT          No such file or directory

That does not translate to `errNoEntry` to me. There is no directory entry, so it would need to be spelled out a little more.

In the end it is private, so I do not mind too much. But I am sure I will wonder about permission errors before assuming the file or directory does not exist.

Reply:
So you prefer `errNoEnt`? ;-)